vignettes/model_definition.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "Mathematical methods for `baselinenowcast`"
+description: "Description of mathematical methods used in the package"
+author: Kaitlyn Johnson
+output:
+  bookdown::html_document2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+bibliography: library.bib
+csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Mathematical methods for baselinenowcast}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# `baselinenowcast` mathematical model
+
+The following describes the estimate of the delay distribution, the generation of the point nowcast, and the estimate of the observation error, for a partially observed or complete reporting triangle. The method assumes that the units of the delays, $d$\$ and the units of reference time $t$ are the same, e.g. weekly data and weekly releases or daily data with daily releases. This method is based on the method described by ([Wolffram et al. 2023](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011394)) developed by the Karlsruhe Institute of Technology.
+
+### Notation
+
+We denote $X_{t,d}, d = 0, .., D$ as the number of cases occurring on time $t$ which appear in the dataset with a delay of $d$. For example, a delay $d = 0$ means that a case occurring on day $t$ arrived in the dataset on day $t$, or on what is considered to be the first possible report date in practice. We only consider cases reporting within a maximum delay $D$. The number of cases reporting for time $t$ with a delay of at most $d$ can be written as:
+
+$$X_{t, \le d} = \sum_{i=0}^d X_{t,i} $$
+
+Such that $X_t = X_{t, \le D}$ is the “final” number of reported cases on time $t$. Conversely, for $d < D$
+
+$$X_{t,>d} = \sum_{i = d+1} ^{D} X_{t,i}$$
+
+is the number of cases still missing after $d$ delays.
+We refer to $X_t$ to describe a random variable, $x_t$ for the corresponding observation, and $\hat{x}_t$ for an estimated/imputed value.
+We refer to the matrix of $x$ available at a given data release time $t^*$ , as the reporting triangle. Here, all $t+d>t^*$ have yet to be observed.
+
+|   | $d = 0$ | $d = 1$ | $d=2$ | $...$ | $d= D-1$ | $d= D$ |
```

Comment:
I really like this. For the actual paper or maybe a future version of this we might want a more abstract schematic as discussed f2f.

Reply:
Yeah I will leave this for now but it might be redundant from the schematic figure...